bindinfo/capture_serial_test.go

```diff
@@ -381,7 +381,7 @@ func TestUpdateSubqueryCapture(t *testing.T) {
 	tk.MustExec("admin capture bindings")
 	rows := tk.MustQuery("show global bindings").Rows()
 	require.Len(t, rows, 1)
-	bindSQL := "UPDATE /*+ use_index(@`upd_1` `test`.`t1` `idx_b`), use_index(@`sel_1` `test`.`t2` ), hash_join(@`upd_1` `test`.`t1`), use_index(@`sel_2` `test`.`t2` )*/ `test`.`t1` SET `b`=1 WHERE `b` = 2 AND (`a` IN (SELECT `a` FROM `test`.`t2` WHERE `b` = 1) OR `c` IN (SELECT `a` FROM `test`.`t2` WHERE `b` = 1))"
+	bindSQL := "UPDATE /*+ use_index(@`upd_1` `test`.`t1` `idx_b`), hash_join(@`upd_1` `test`.`t1`)*/ `test`.`t1` SET `b`=1 WHERE `b` = 2 AND (`a` IN (SELECT /*+ use_index(@`sel_1` `test`.`t2` )*/ `a` FROM `test`.`t2` WHERE `b` = 1) OR `c` IN (SELECT /*+ use_index(@`sel_2` `test`.`t2` )*/ `a` FROM `test`.`t2` WHERE `b` = 1))"
```

Comment:
Why this change? Is it expected?

Reply:
yes,this pr is place hint at owner's query block rather than not place all hint in main query block